### ISSUE TITLE
source code release

### ISSUE THREAD

**Issue by xjsxujingsong** [benjiebob/BADJA#2]
Hi do you have any plan to public the code described in the paper? Thanks.

**Comment by benjiebob** [comment]
Yes, definitely. We plan to release the code + a newer end-to-end version + a large annotated dataset very soon, when we get the results for a recent submission. 

If your need is urgent, I'd recommend looking at the SMPLify code [http://smplify.is.tue.mpg.de/] which optimizes the SMPL human mesh model to images. Adapting this to the SMAL mesh is the basis for our optimization step.

Stay tuned! 
Ben

**Comment by xjsxujingsong** [comment]
Thanks for your reply. I am just starting to understand how smpl, smal, smalst work. Looking forward to your code.

**Comment by buaaswf** [comment]
Dear @xjsxujingsong ,
Thanks for your sharing, could you please release the testing code firstly, I want to apply your work to a new work.
Thanks very much,

yours
Wenfeng
